docs/source/deployment/databricks/databricks_deployment_workflow.md

```diff
@@ -95,6 +95,10 @@ kedro new --starter=databricks-iris
 
 This command creates a new Kedro project using the `databricks-iris` starter template. Name your new project `iris-databricks` for consistency with the rest of this guide.
 
+ ```{note}
+ If you are not using the `databricks-iris` starter to create a Kedro project, then you should [disable file-based logging](../../logging/index.md#disable-file-based-logging) to prevent Kedro from attempting to write to the read-only file system.
```

Comment:
```suggestion
 If you are not using the `databricks-iris` starter to create a Kedro project, **and** you are working with a version of Kedro **earlier than 0.19.0**, then you should [disable file-based logging](../../logging/index.md#disable-file-based-logging) to prevent Kedro from attempting to write to the read-only file system.
```

Reply:
Even if we keep this note, the link is broken since the logging docs now have "enable file based logging" section in `develop`. The docs build is failing.

Reply:
Good point @ankatiyar but we can resolve that by pointing to the 0.18.14 version of the docs as the latest version for that specific issue. Let me do that fix now...